casaos-healthcheck/run.sh

```
@@ -381,6 +381,325 @@ check_dmesg_errors() {
+        if [ "$swap_percent" -gt 80 ]; then
+            print_color "0;31" "${CROSS_MARK} High swap usage: ${swap_percent}%"
+        else
+            print_color "0;32" "${CHECK_MARK} Swap usage: ${swap_percent}%"
+        fi
+    fi
+    echo "Swappiness value: $swappiness"
+    
+    # Check memory pressure stats if available
+    if [ -f "/proc/pressure/memory" ]; then
+        echo -e "\nMemory Pressure Statistics:"
+        local pressure=$(cat /proc/pressure/memory)
```

Shellcheck

[warning] Declare and assign separately to avoid masking return values.

(SC2155)